.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ros_distribution: [melodic, noetic, galactic, humble, iron, rolling]
+        ros_distribution: [melodic, noetic, humble, iron, rolling]
```

Comment:
maybe we should leave that? Technically people could still manually build for `galactic`